cmd/login/runner.go

```diff
@@ -415,7 +415,7 @@ func (r *runner) createClusterClientCert(ctx context.Context) error {
 	}
 
 	// Store client certificate credential into the kubeconfig.
-	contextName, contextExists, err := storeCredential(r.k8sConfigAccess, r.fs, clientCertResource, secret, clusterBasePath)
+	contextName, contextExists, err := storeCredential(r.k8sConfigAccess, r.fs, r.flag.WCSelfContained, clientCertResource, secret, clusterBasePath)
```

Comment:
Is it possible to have an if statement here where we decide whether we `store` or `print` the credential? It's a bit confusing to have printing done in the `storeCredential` function

Reply:
Sure, we can do that, too! :)